atomata/capstone_WebClient
# Testingsprint4

### Summary 
Modified some unit test to follow current code

### Details
Add getjson test to test wether it get a feed back.
The main change of this pr is in package.json and some refactoring. 

### How to test?
yarn install --dev
yarn test 
all test shall pass

yarn dev
make sure that the program works the way is was.
